server/routes/user.js

```diff
@@ -4,17 +4,55 @@ const User = require("../models/User");
 
 // Find user
 router.get("/", async (req, res, next) => {
-  console.log(req.body)
+  console.log(req.body);
   const { id } = req.body;
   try {
     const user = await User.findUser(id);
-    res.status(201).json({ msg: user });
-    }
-  catch (err) {
+    res.status(201).send(user);
+  } catch (err) {
+    res.status(404).send({ msg: err });
+  }
+});
+
+// Find user by email
+router.post("/", async (req, res, next) => {
```

Comment:
It's kind of weird to make this a POST route since you're getting information. Would make more sense as a GET route to a path like "/find-by-email" and use a req.query instead of body. So a request would be to "/find-by-email?email={}"

Reply:
I was having trouble receiving what I wanted using "get", so I resorted to "post". I now realize that I had 2 get('/'.. in the user.js route and that was the source of problem.  I added  get("/<name> ..